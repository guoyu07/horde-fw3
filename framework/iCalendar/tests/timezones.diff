003- 14:00
003+ 
004- Lady Aibell Chat
004+ Fatal error: Call to undefined method Horde_Date::format() in /home/wrobel/usr/devel/gh/horde-glue/horde-fw3/framework/iCalendar/tests/timezones.php on line 22
005- 15:00
006- Lady Aibell Chat
007- 15:00
008- FAR Sexy Sunday Chat
009- 14:00
010- Sapphire Phelan's Birthday and She Wants to Party Chat
011- 01:00
012- Bianca's Chat
013- 15:00
014- Lady Aibell Chat @ Coffeetime Devin Group
015- 15:00
016- Live Editor Chat
017- 00:00
018- FAR Sexy Sunday Chat
019- 14:00
020- Tammy Lee Author Live Chat
021- 18:00
022- CPLLC Romance Authors @ Coffeetime
023- 15:00
024- CPLLC Romance Authors @ Coffeetime
025- 15:00
026- Ella Scopilo's Hump Day chat
027- 01:00
028- 
029- MMMPseminar.ics
030- Jack Murphy
031- Trojan Horse or Proton Force: Finding the Right Partners for Toxin Translocation
032- 18:00
033- Kai Matuschewski
034- Hitting the Plasmodium Life Cycle Early On: Attenuated Liver Stages
035- 17:00
036- Felix Rey
037- Insights into the Mechanism of Membrane Fusion Derived from Structural Studies of Viral Envelope Proteins
038- 18:00
039- Maurizio Del Poeta
040- Sphingolipid-Mediated Fungal Pathogenesis
041- 18:00
042- Jorge Galan
043- Structure, Assembly, and Function of the Type III Secretion Injectisome
044- 18:00
045- Don Ganem
046- RNAi, MicroRNAs and Viral Infection
047- 18:00
048- Barak Cohen
049- Genomic Analysis of Natural Variation in Saccharomyces
050- 18:00
051- Michael Ferguson
052- The Structure and Biosynthesis of Trypanosome Surface Molecules:  Basic Science and Therapeutic Possiblities
053- 18:00
054- James M. Musser
055- Molecular Pathogenomics of Group A Streptococcus, the Flesh-Eater
056- 17:00
057- Wayne Yokoyama
058- Innate Responses to Viral Infections
059- 17:00
060- Herbert \"Skip\" Virgin
061- Host-Herpesvirus Standoff:  Good News From the Front in an Ancient Battle
062- 18:00
063- New Event
064- 17:00
065- Matthew Welch
066- Exploitation of the Host Actin Cytoskeleton by Bacterial and Viral Pathogens
067- 17:00
068- Theresa Koeher
069- Virulence Gene Expression by Bacillus anthracis and Implications for the Host
070- 17:00
071- Eduardo Groisman
072- Regulatory Networks Controlling Bacterial Physiology and Virulence
073- 18:00
074- Brendan Cormack
075- Transcriptional Silencing and Adherence in the Yeast Pathogen Candida glabrata
076- 17:00
077- Andrew Pekosz
078- Intracellular Transport of Viral Proteins and Particles
079- 18:00
080- 
081- Moon_Days.ics
082- Standard Time resumes
083- 07:00
084- New Moon
085- 16:05
086- Full Moon
087- 03:13
088- Full Moon
089- 12:58
090- Full Moon
091- 00:25
092- Autumnal Equinox
093- 04:03
094- New Moon
095- 22:18
096- Full Moon
097- 03:02
098- New Moon
099- 11:45
100- New Moon
101- 20:10
102- Full Moon
103- 22:54
104- Full Moon
105- 18:42
106- New Moon
107- 14:01
108- Full Moon
109- 13:57
110- Summer Solstice
111- 12:26
112- Full Moon
113- 05:45
114- New Moon
115- 04:01
116- New Moon
117- 04:31
118- New Moon
119- 05:14
120- 
121- ProjectCalendar.ics
122- Code slush begins
123- 03:00
124- String freeze
125- 03:00
126- Calendar Test Day
127- 12:00
128- Calendar QA Chat
129- 16:30
130- Status Meeting
131- 16:00
132- 
133- SpanishHolidays.ics
134- 
135- allcategories.vcs
136- NSS ISDC 2006
137- 04:00
138- VSE Trailer Ohio State Tour
139- 12:00
140- AIAA International Energy Conversion Conference
141- 04:00
142- 42nd AIAA/ASME/SAE/ASEE Joint Propulsion Conference
143- 04:00
144- 2006 Beam Power Challenge
145- 04:00
146- 2006 Tether Challenge
147- 04:00
148- X PRIZE Cup Spaceflight Exposition
149- 04:00
150- 2007 Astronaut Glove Challenge: APRIL DATE TBD
151- 05:00
152- 2007 Regolith Excavation Challenge
153- 04:00
154- 2007 Personal Air Vehicle (PAV) Challenge: DATE TBD
155- 04:00
156- 2008 MoonROx (Moon Regolith Oxygen) Challenge
157- 04:00
158- 
159- arsenal32FC.ics
160- Sheffield Utd (2)0-0(4) Arsenal
161- 20:05
162- Arsenal 5-3 Middlesbrough
163- 15:05
164- West Ham Utd 0-0 Arsenal
165- 14:00
166- Arsenal 2-1 Tottenham
167- 15:00
168- Birmingham City 2-1 Arsenal
169- 14:00
170- Carling Cup Final
171- 15:00
172- Fulham 0-3 Arsenal
173- 14:00
174- Leeds United 1-4 Arsenal
175- 16:05
176- Barcelona 2-1 Arsenal
177- 18:45
178- Manchester City 1-3 Arsenal
179- 18:45
180- Arsenal 1-1 Aston Villa
181- 14:00
182- River Plate 0-0 Arsenal
183- 17:00
184- Weiz 0-5 Arsenal
185- 17:00
186- Boreham Wood 0-4 Arsenal XI
187- 18:30
188- Carling Cup Semi-Final (2)
189- 20:00
190- Arsenal - Liverpool
191- 16:00
192- Arsenal - Newcastle Utd
193- 15:00
194- Arsenal 2-1 Bolton Wanderers
195- 15:00
196- Charlton Athletic 1-3 Arsenal
197- 15:00
198- F.A. Cup 4rth round
199- 15:00
200- Southampton 1-1 Arsenal
201- 12:45
202- Ritzing 2-5 Arsenal
203- 14:00
204- Arsenal 3-0 Portsmouth
205- 15:00
206- Beveren 0-0 Arsenal XI
207- 17:30
208- Wolverhampton 1-3 Arsenal
209- 15:00
210- Fulham - Arsenal
211- 19:45
212- Hamburg - Arsenal
213- 18:45
214- Champions League knockout round 2
215- 19:45
216- Arsenal 3-2 Newcastle United
217- 19:00
218- Arsenal 1-1 Manchester United
219- 15:05
220- Arsenal 2-1 Manchester City
221- 16:05
222- Arsenal 0-0 AFC Ajax
223- 19:45
224- Arsenal 2-0 Lokomotiv Moscow
225- 19:45
226- Arsenal (5)0-0(4) Manchester Utd
227- 14:00
228- Lokomotiv Moscow 0-0 Arsenal
229- 16:30
230- Arsenal 2-1 Chelsea
231- 14:00
232- Manchester Utd 0-0 Arsenal
233- 15:05
234- St Albans City 1-3 Arsenal XI
235- 18:30
236- Chelsea 1-0 Arsenal
237- 15:00
238- Manchester City 1-0 Arsenal
239- 16:15
240- Arsenal 0-0 Manchester Utd
241- 20:00
242- Blackburn Rovers 1-0 Arsenal
243- 15:00
244- FC Porto - Arsenal
245- 19:45
246- Bolton W 1-1 Arsenal
247- 15:00
248- Aston Villa - Arsenal
249- 15:00
250- Real Madrid 0-1 Arsenal
251- 19:45
252- Arsenal 1-1 West Bromwich Albion
253- 15:00
254- Bolton Wanderers 2-0 Arsenal
255- 15:00
256- Manchester Utd - Arsenal
257- 15:00
258- Arsenal - Reading
259- 15:00
260- Arsenal 1-0 Manchester City
261- 14:00
262- Arsenal - Chelsea
263- 14:00
264- Champions League knockout round 2
265- 19:45
266- AZ Alkmaar 0-3 Arsenal
267- 17:00
268- Everton 1-4 Arsenal
269- 13:00
270- Readling - Arsenal
271- 15:00
272- Middlesbrough 2-1 Arsenal
273- 16:15
274- Barnet 1-4 Arsenal
275- 14:00
276- Champions League semi-final 1
277- 18:45
278- CSKA Moscow - Arsenal
279- 16:30
280- Arsenal 2-1 Ajax
281- 16:00
282- Arsenal 1-0 Birmingham City
283- 12:30
284- Arsenal 2-2 Southampton
285- 14:00
286- Everton - Arsenal
287- 15:00
288- Schwadorf 1-8 Arsenal
289- 15:00
290- Celta de Vigo 2-3 Arsenal
291- 19:45
292- Sheffield Utd - Arsenal
293- 15:00
294- Arsenal 3-1 Everton
295- 19:45
296- Arsenal 2-0 Newcastle
297- 12:30
298- Arsenal 3-0 Charlton Athletic
299- 15:00
300- Arsenal - Charlton Athletic
301- 15:00
302- Arsenal - West Ham
303- 14:00
304- Arsenal 2-1 Chelsea
305- 11:30
306- Portsmouth 0-1 Arsenal
307- 16:05
308- Arsenal 0-0 Real Madrid
309- 19:45
310- F.A. Cup Semi-Final
311- 14:00
312- Champions League semi-final 1
313- 18:45
314- Arsenal 0-0 Fulham
315- 14:00
316- Arsenal 2-1 Leicester City
317- 14:00
318- Arsenal 3-1 Sunderland
319- 15:00
320- Arsenal 1-0 Dynamo Kyiv
321- 19:45
322- FC Utrecht 0-3 Arsenal
323- 16:00
324- Chelsea 1-1 Arsenal
325- 19:45
326- Manchester City 1-2 Arsenal
327- 15:05
328- Man Utd 1-0 Arsenal
329- 19:45
330- Arsenal 1-0 Blackburn Rovers
331- 14:00
332- Wigan Athletic - Arsenal
333- 19:45
334- Arsenal 3-0 Sparta Prague
335- 19:45
336- Arsenal 1-1 Portsmouth
337- 14:00
338- Arsenal 2-1 Dinamo Zagreb
339- 19:05
340- Birmingham City 0-3 Arsenal
341- 15:00
342- Champions League quarter-final 1
343- 18:45
344- Barnet 0-0 Arsenal
345- 14:00
346- Arsenal 2-2 Bolton Wanderers
347- 11:45
348- Arsenal - Middlesbrough
349- 14:00
350- Dynamo Kyiv 2-1 Arsenal
351- 18:45
352- Arsenal 3-1 Aston Villa
353- 14:00
354- Arsenal - Watford
355- 14:00
356- Portsmouth 1-1 Arsenal
357- 19:00
358- Sparta Prague 0-2 Arsenal
359- 18:45
360- Bolton 0-1 Arsenal
361- 12:15
362- Arsenal 1-1 Bolton Wanderers
363- 15:00
364- Arsenal 3-0 Blackburn Rovers
365- 15:00
366- Arsenal 3-1 ManUtd
367- 14:00
368- Champions League semi-final 2
369- 18:45
370- Arsenal - Wigan Athletic
371- 15:00
372- Sunderland 0-3 Arsenal
373- 16:15
374- Arsenal - Tottenham
375- 12:45
376- Villarreal CF 0-0 Arsenal
377- 18:45
378- Arsenal - Fulham
379- 14:00
380- Arsenal 1-2 Chelsea
381- 18:45
382- F.A. Cup 5th round
383- 15:00
384- Arsenal 3-0 Reading
385- 19:45
386- Arsenal 3-0 Blackburn Rovers
387- 11:15
388- Liverpool 1-0 Arsenal
389- 20:00
390- Ajax 0-0 Arsenal
391- 18:45
392- Bolton Wanderers 1-0 Arsenal
393- 17:15
394- Arsenal 2-1 Wigan Athletic
395- 19:45
396- Aston Villa 0-0 Arsenal
397- 12:45
398- Dinamo Zagreb 0-3 Arsenal
399- 19:05
400- Newcastle Utd - Arsenal
401- 14:00
402- Grazer AK 1-2 Arsenal
403- 16:00
404- Carling Cup Semi-Final (1)
405- 20:00
406- Everton 1-0 Arsenal
407- 12:45
408- Porto 1-2 Arsenal
409- 16:30
410- Bayern München 3-1 Arsenal
411- 19:45
412- Arsenal 4-2 Liverpool
413- 11:30
414- Arsenal 2-1 Liverpool
415- 16:00
416- Barnet 1-10 Arsenal
417- 14:00
418- Arsenal - Portsmouth
419- 15:00
420- Watford - Arsenal
421- 17:30
422- Arsenal 1(9)-1(8) Rotherham United
423- 19:45
424- Manchester Utd 2-0 Arsenal
425- 15:05
426- Aston Villa 0-2 Arsenal
427- 14:00
428- Boreham Wood 2-6 Arsenal XI
429- 18:30
430- Champions League quarter-final 2
431- 18:45
432- Bolton 1-0 Arsenal
433- 17:40
434- Champions League semi-final 2
435- 18:45
436- West Bromwich Albion 0-2 Arsenal
437- 19:00
438- Bolton Wanderers - Arsenal
439- 17:15
440- Arsenal 1-1 Sheffield Utd
441- 12:30
442- Arsenal - Blackburn Rovers
443- 15:00
444- Carling Cup 3rd round
445- 19:00
446- Arsenal 5-0 Aston Villa
447- 14:00
448- Sunderland 0-3 Arsenal
449- 18:45
450- Arsenal 2-1 Charlton Athletic
451- 15:00
452- Arsenal 4-1 Middlesbrough
453- 15:00
454- Arsenal 1-0 Newcastle
455- 16:05
456- Arsenal 2-0 Fulham
457- 13:00
458- KSK Beveren 2-2 Arsenal XI
459- 16:00
460- Juventus 0-0 Arsenal
461- 18:45
462- Arsenal 4-2 Wigan Athletic
463- 14:00
464- Newcastle Utd 0-1 Arsenal
465- 20:00
466- Manchester City 0-1 Arsenal
467- 14:00
468- Middlesbrough 0-4 Arsenal
469- 14:00
470- Champions League knockout round 1
471- 19:45
472- Middlesbrough 0-4 Arsenal
473- 15:05
474- Everton 1-1 Arsenal
475- 20:00
476- Portsmouth - Arsenal
477- 14:00
478- Tottenham Hotspur 1-1 Arsenal
479- 12:00
480- Champions League quarter-final 2
481- 18:45
482- Arsenal 3-0 Birmingham City
483- 15:00
484- Arsenal 3-1 West Bromwich Albion
485- 14:00
486- Leeds United 1-4 Arsenal
487- 15:00
488- Arsenal 2-2 Chelsea
489- 16:05
490- Carling Cup 4th round
491- 20:00
492- NK Maribor 2-3 Arsenal
493- 17:30
494- Barnet 0-0 Arsenal
495- 14:00
496- Wigan Athletic 1-0 Arsenal
497- 19:45
498- Norwich City 1-4 Arsenal
499- 16:15
500- Portsmouth 1-5 Arsenal
501- 18:00
502- Blackburn Rovers 0-2 Arsenal
503- 15:00
504- Tottenham - Arsenal
505- 14:00
506- Liverpool - Arsenal
507- 14:00
508- Arsenal 2-1 FC Thun
509- 18:45
510- Blackburn Rovers - Arsenal
511- 15:00
512- Arsenal 3-0 Blackburn Rovers
513- 18:45
514- Arsenal 2-0 Aston Villa
515- 18:45
516- Middlesbrough - Arsenal
517- 15:00
518- Arsenal 2-1 Cardiff City
519- 13:00
520- Arsenal 3-1 Liverpool
521- 15:05
522- Champions League quarter-final 1
523- 18:45
524- Arsenal 0-1 Manchester Utd
525- 11:00
526- Middlesbrough 0-1 Arsenal
527- 14:00
528- FC Thun 0-1 Arsenal
529- 19:45
530- Panathinaikos 2-2 Arsenal
531- 18:45
532- West Ham Utd - Arsenal
533- 13:30
534- Charlton Athletic 1-1 Arsenal
535- 14:00
536- Arsenal - Everton
537- 14:00
538- Arsenal 7-0 Middlesbrough
539- 15:00
540- Newcastle Utd 0-0 Arsenal
541- 15:05
542- PSV Eindhoven 1-1 Arsenal
543- 19:45
544- Charlton Athletic - Arsenal
545- 14:00
546- Arsenal 2-0 Juventus
547- 18:45
548- F.A. Cup 6th round
549- 15:00
550- Arsenal - Hamburg
551- 19:45
552- Portsmouth 1-1 Arsenal
553- 19:00
554- West Bromwich Albion 0-2 Arsenal
555- 20:00
556- Champions League knockout round 1
557- 19:45
558- Arsenal 1-1 Tottenham Hotspur
559- 11:45
560- Arsenal 2-1 Stoke City
561- 14:00
562- Birmingham City 0-2 Arsenal
563- 15:00
564- Arsenal 2-0 Southampton
565- 19:45
566- Arsenal 2-3 West Ham Utd
567- 20:00
568- Arsenal 0-2 Chelsea
569- 16:00
570- Southampton 0-1 Arsenal
571- 20:00
572- Arsenal 1-1 Manchester City
573- 19:45
574- Tottenham 2-2 Arsenal
575- 15:05
576- Leicester City 1-1 Arsenal
577- 15:00
578- Arsenal 4-1 Middlesbrough
579- 15:00
580- Tottenham Hotspur 4-5 Arsenal
581- 12:00
582- Arsenal 1-0 Tottenham Hotspur
583- 19:00
584- Newcastle Utd 1-0 Arsenal
585- 17:15
586- Arsenal 1(3)-1(4) ManUtd
587- 13:00
588- Inter Milan 1-5 Arsenal
589- 19:45
590- West Bromwich Albion 2-1 Arsenal
591- 14:00
592- Blackburn Rovers 0-1 Arsenal
593- 12:45
594- Manchester Utd 2-0 Arsenal
595- 15:00
596- Austria Vienna 0-2 Arsenal
597- 18:00
598- Arsenal 2-1 Everton
599- 14:00
600- Arsenal 3-0 Wolverhampton W
601- 12:00
602- AFC Ajax 1-2  Arsenal
603- 18:45
604- Arsenal - Sheffield Utd
605- 14:00
606- Manchester City 1-2 Arsenal
607- 18:45
608- Arsenal 7-0 Everton
609- 19:00
610- Crystal Palace 1-1 Arsenal
611- 17:15
612- Arsenal 1-0 Besiktas
613- 17:00
614- Arsenal - FC Porto
615- 18:45
616- Rangers 0-3 Arsenal
617- 18:45
618- Liverpool 2-1 Arsenal
619- 16:05
620- Arsenal 1-0 PSV Eindhoven
621- 18:45
622- Liverpool 1-2 Arsenal
623- 11:30
624- Arsenal - Manchester Utd
625- 15:00
626- Charlton Athletic 0-1 Arsenal
627- 12:45
628- Arsenal 1-0 Bayern München
629- 19:45
630- Arsenal 5-1 Rosenborg
631- 19:45
632- Chelsea 2-1 Arsenal
633- 14:00
634- Champions League Final
635- 18:45
636- Arsenal 0-3 Inter Milan
637- 18:45
638- Fulham 0-1 Arsenal
639- 15:05
640- Arsenal 0-1 Middlesbrough
641- 19:45
642- Arsenal - CSKA Moscow
643- 19:45
644- Arsenal 1-1 Panathinaikos
645- 19:45
646- Chelsea - Arsenal
647- 16:00
648- Aston Villa 1-3 Arsenal
649- 17:15
650- Arsenal 2-0 Celta de Vigo
651- 19:45
652- Arsenal 5-0 Leeds
653- 19:00
654- Arsenal 5-1 Crystal Palace
655- 20:00
656- Fulham 0-4 Arsenal
657- 15:00
658- Arsenal 4-0 Portsmouth
659- 19:45
660- Arsenal - Manchester City
661- 19:45
662- Ritzing 2-2 Arsenal
663- 18:00
664- Arsenal 2-4 Manchester Utd
665- 20:00
666- Arsenal - Bolton Wanderers
667- 14:00
668- Arsenal 1-0 Villarreal CF
669- 18:45
670- Arsenal 5-1 Wolverhampton W.
671- 19:45
672- Celtic 1-1 Arsenal
673- 14:00
674- F.A. Cup Final
675- 14:00
676- Arsenal 2-0 Wolverhampton Wanderers
677- 15:00
678- Peterborough 1-0 Arsenal XI
679- 18:30
680- Arsenal 2-0 Everton
681- 19:00
682- Arsenal 4-0 Charlton Athletic
683- 14:00
684- Arsenal 4-1 Fulham
685- 18:45
686- Chelsea 0-0 Arsenal
687- 19:00
688- Doncaster Rovers 2-2P Arsenal
689- 19:45
690- Sturm Graz 0-2 Arsenal
691- 16:30
692- Rosenborg 1-1 Arsenal
693- 18:45
694- Arsenal 6-0 England XI
695- 18:45
696- Arsenal 4-1 Norwich City
697- 14:00
698- Arsenal 0-0 Birmingham City
699- 11:30
700- Middlesbrough 2-1 Arsenal
701- 20:00
702- Ajax 0-1 Arsenal
703- 19:15
704- Chelsea 1-2 Arsenal
705- 12:30
706- Boreham Wood 1-1 Arsenal XI
707- 18:45
708- Carling Cup 5th round
709- 20:00
710- KSK Beveren 3-3 Arsenal XI
711- 18:30
712- Wigan Athletic 2-3 Arsenal
713- 12:45
714- F.A. Cup 3rd round
715- 15:00
716- SV Mattersburg 1-2 Arsenal
717- 17:00
718- 
719- events.ics
720- test cet 2
721- 13:00
722- 
723- exchange.ics
724- internal final review of mmc site changes
725- 19:00
726- 
727- exdate.ics
728- 
729- iscw.ics
730- Ontoweb Day http://nextwebgeneration.com/meetings/ontoweb5/
731- 13:30
732- Ontoweb SIG day http://nextwebgeneration.com/meetings/ontoweb5/
733- 13:30
734- Beach Barbeque
735- 00:00
736- 
737- meeting.ics
738- Updated: Webex Training - Encryption Push
739- 17:00
740- 
741- privacy_events.ics
742- Australian Smart Card Summit
743- 04:00
744- SOCAP Australia 2006 Symposium
745- 03:30
746- 
747- rfc2445.ics
748- Bastille Day Party
749- 17:00
750- Annual Employee Review
751- 16:30
752- Laurel is in sensitivity awareness class.
753- 16:30
754- Our Blissful Anniversary
755- 00:00
756- XYZ Project Review
757- 13:30
758- Calendaring Interoperability Planning Meeting
759- 12:30
760- 
761- test.vcs
762- 2007 Astronaut Glove Challenge: APRIL DATE TBD
763- 05:00
764- 
765- test4.vcs
766- Chiefs vs. Buffalo @ Arrowhead Stadium
767- 19:00
768- 
769- test_recurring.vcs
770- XXXStriked out XXX
771- 09:00
772- 
773- wicca.ics
774- Wolf Moon
775- 10:32
776- Snow Moon
777- 04:54
778- Worm Moon
779- 20:58
780- Pink Moon
781- 10:06
782- Flower Moon
783- 20:18
784- Strawberry Moon
785- 04:14
786- Buck Moon
787- 11:00
788- Sturgeon Moon
789- 17:53
790- Harvest Moon
791- 02:01
792- Hunter's Moon
793- 12:14
794- Beaver Moon
795- 00:57
796- Cold Moon
797- 16:15