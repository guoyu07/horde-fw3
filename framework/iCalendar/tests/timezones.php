<?php

require_once dirname(dirname(__FILE__)) . '/iCalendar.php';
require_once 'Horde/Date.php';
putenv('TZ=UTC');

$test_files = glob(dirname(__FILE__) . '/fixtures/vTimezone/*.???');
foreach ($test_files as $file) {
    echo basename($file) . "\n";
    $ical = new Horde_iCalendar();
    $ical->parsevCalendar(file_get_contents($file));
    foreach ($ical->getComponents() as $component) {
        if ($component->getType() != 'vEvent') {
            continue;
        }
        $date = $component->getAttribute('DTSTART');
        if (is_array($date)) {
            continue;
        }
        echo $component->getAttribute('SUMMARY') . "\n";
        $d = new Horde_Date($date);
        echo $d->format('H:i') . "\n";
    }
    echo "\n";
}

?>
