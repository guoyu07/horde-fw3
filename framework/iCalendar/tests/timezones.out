AuthorChats.ics
FAR Sexy Sunday Chat

Fatal error: Call to undefined method Horde_Date::format() in /home/wrobel/usr/devel/gh/horde-glue/horde-fw3/framework/iCalendar/tests/timezones.php on line 22